src/neptune/new/types/atoms/file_types.py

```diff
@@ -0,0 +1,170 @@
+#
```

Comment:
Remove all these `_` prefixes and move it to `internal` package. Everything that is outside of `internal` should be treated as a user-exposed

Reply:
Good point.